docker/pvs-studio.Dockerfile

```
@@ -19,3 +40,34 @@ RUN apt-get update \
+    && pvs-studio-dotnet --version
+
+# Install .NET SDK via Microsoft's install script, AFTER the pvs-studio apt
+# packages, so this SDK (10.0.400) is the one on PATH and under DOTNET_ROOT.
+# --channel 10.0.4xx selects the latest patch in the 10.0.4xx feature band
+# (e.g. 10.0.400), matching global.json's version 10.0.400 + rollForward latestFeature.
+#
+# The install script is verified before running: it is downloaded alongside
+# Microsoft's detached GPG signature and public key, the key fingerprint is
+# pinned to the known-good Microsoft signing key, and the script must verify
+# against that key or the build fails.
+RUN set -euo pipefail \
```

Hadolint (2.15.1)

[warning] Either use Wget or Curl but not both

(DL4001)